docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
-      print output
-      ```
+> entity_type: text
 
-    - *CURL command:*
+The Text Detector has the capability to detect custom text entity within the given text.  The detector has the ability to handle multilanguage text. 
 
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+> To create new text type entities, follow the steps from [here](https://github.com/hellohaptik/chatbot_ner/blob/develop/docs/adding_entities.md).
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/location/?message=atm+in+andheri+west&entity_name=locality_list&structured_value=&fallback_value=&bot_message='
-      ```
+**API Examples:**
 
-    - *CURL Output:*
+- **Example 1: *Detecting text entity from message***
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "message", 
-            "original_text": "andheri west", 
-            "entity_value": {
-              "value": "Andheri West"
-            }, 
-            "language": "en"
-          }
-        ]
-      }
-      ```
+  - *Django Shell:* 
 
-### person_name
+    ```python
+    >>> message='i want to order chinese from  mainland china and pizza from domminos'
+    >>> entity_name='restaurant'
+    >>> structured_value=None
+    >>> fallback_value=None
+    >>> bot_message=None
+    >>> source_language='en'
+    
+    >>> from ner_v1.chatbot.entity_detection import get_text
+    >>> output = get_text(message=message, entity_name=entity_name,
+                          structured_value=structured_value,
+                          fallback_value=fallback_value,
+                          bot_message=bot_message,language=source_language)
+    >>> print(output)
+    
+    ```
 
-- This functionality calls the NameDetector class to detect Name entities.
+    The above can also be done from within the Docker container's shell. Setup is in docker.md file.
 
-- Examples:
+  - *CURL command:*
 
-  - Example 1:
+    ```shell
+    URL='localhost'
+    PORT=8081
+    
+    curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20want%20to%20order%20chinese%20from%20%20mainland%20china%20and%20pizza%20from%20domminos&entity_name=restaurant&structured_value=&fallback_value=None&bot_message=&source_language=en'
+    
+    ```
 
-    - ```python
-      message = 'my name is yash doshi'
-      entity_name = 'person_name'
-      structured_value = None
-      fallback_value = 'Guest'
-      bot_message = 'what is your name ?'
-      ```
+    > **Output **:
 
-    - *Python:*
+    ```json
+    {"data": [
+        {
+            "detection": "message",
+            "original_text": "mainland china",
+            "entity_value": {"value": "Mainland China"},
+            "language": "en"
+        },
+        {
+            "detection": "message",
+            "original_text": "dominos",
+            "entity_value": { "value": "Domino's Pizza"},
+            "language": "en"
+        }]
+    }
+    
+    ```
+
+
+
+- **Example 2: *Detecting text entity from structured value***
```

Comment:
Remove this example and use 'city' example in hindi instead. Also index city dictionary in hindi on startup. (just add in data folder)

Reply:
Would that work? Does the csv reading code handle language correctly ?